src/blocks/frontend/live-search/style.scss

```diff
@@ -0,0 +1,67 @@
+.o-live-search {
+	.wp-block-search__inside-wrapper {
+		position: relative;
+	}
+
+	.search-results {
+		position: absolute;
+		z-index: 5;
+
+		background-color: var( --formfieldbgcolor, #fff );
+		border-radius: var( --formfieldborderradius, 1px );
```

Comment:
If this is a variable from us.  I recommend using a dash in the name. CSS vars are always a pain when it comes to typos.

`--formfieldbgcolor`

The difference is very clear.

`--form-field-bg-color`

Reply:
This are vars from Neve. I thought it would be nice to have the same style as the input field, at least when used with Neve

Reply:
Then we can keep them, but a comment to mention that they're for inheriting Neve values so we might not change them without knowing 🙈 .